src/home_robot/home_robot/agent/objectnav_agent/objectnav_agent.py

```diff
@@ -362,7 +367,8 @@ def _preprocess_obs(self, obs: Observations):
             if self.verbose:
                 print("object goal =", obs.task_observations["object_goal"])
             object_goal_category = torch.tensor(
-                obs.task_observations["object_goal"]
+                # obs.task_observations["object_goal"]
```

Comment:
I've hardcoded the objectnav agent to use only 5 channels (object goal, start receptacle, end receptacle, and 2 'other' channels) since we'll never need the full set of objects.

Reply:
I've assigned these channel values to variables like we discussed (not using global constants/macros since they're only used in this one function).